region/norfair/east/Norfair Reserve Tank Room.json

```diff
@@ -151,6 +151,39 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 1],
+      "name": "Collect Both Items with Pause Abuse",
+      "requires": [
+        {"or": [
+          "canBounceBall",
+          "canHorizontalDamageBoost",
+          "Wave",
+          "Plasma",
+          "Spazer"
+        ]},
+        {"enemyKill": {
+          "enemies": [
+            ["Sova", "Sova", "Sova"]
+          ],
+          "explicitWeapons": ["PowerBomb"]
+        }},
+        "canInsaneJump",
+        "canPauseAbuse",
```

Comment:
Talking to Maddo about this one, The pause abuse would also require Samus to end with a small amount of energy (probably 1). Also, we weren't sure what the bounce ball option was for, if it should require lava frames, and maybe be a separate strat?

Reply:
- The strat as it was depended on the right Sova dropping a big energy, and that wasn't reliable enough.  The way it is now you need 2 big energy's across 5 drops, or a right Sova big energy with a pause abuse.  
- The bounceball meant rolling from the middle block  to the left of middle block that has the middle Sova on it.  The movement is a little nicer if you bounce instead of roll, but its not a big difference.

Reply:
Been trying this out for several hours and am able to get it sometimes, but probably only with a bit luckier drops than the strat has in mind. To me the pause abuse never seems helpful. Maybe the strat name and note should be adjusted to account for the the pause abuse being optional and RNG-dependent? If the drops on the left side of the room turn out mediocre while the ones on the right side are really good, then pause abuse makes sense; in those cases I guess I just never managed to get it. If there's some detail I'm missing, a video might help, though I'm guessing it's just a general movement skill issue. Either way, the logic here looks fine.

Reply:
Here's my video of it: https://videos.maprando.com/video/1165